holoviews/plotting/bokeh/graphs.py

```diff
@@ -12,16 +12,24 @@
 from .chart import ColorbarPlot, PointPlot
 from .element import (CompositeElementPlot, LegendPlot, line_properties,
                       fill_properties, text_properties)
-from ..util import process_cmap
+from ..util import process_cmap, get_directed_graph_paths
 from .util import rgba_tuple
 
 
 class GraphPlot(CompositeElementPlot, ColorbarPlot, LegendPlot):
 
+    arrow_length = param.Number(default=0.025, doc="""
+      If directed option is enabled this determines the length of the
+      arrows as fraction of the overall extent of the graph.""")
```

Comment:
The size of the arrow head I assume? I would expect the length of the arrow to be the length of the edge...

Reply:
If this is what you mean, then I'm not so sure about the name...

Reply:
True, it's the arrow head, ``head_length`` doesn't seem that descriptive and ``arrow_head_length`` seems too long. Any other suggestions?

Reply:
Maybe `size` as it is shorter than `length`?

Reply:
I also think `arrow_size` is more obviously associated with the head of the arrow. I think `arrow_size` might be shorter and less confusing...

Reply:
I don't associate `arrow_size` with the head; that seems like the size of some arrow glyph.